src/Login.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { useState, ChangeEvent, FormEvent } from 'react';
+import {
+  Text,
+  Button,
+  TextInput,
+  PasswordInput,
+} from '@mantine/core'
+
+interface FormValues {
+  password: string;
+  email: string;
+  confirmPassword?: string;
+}
+
+export default function Login() {
+  const [isRegistered, setIsRegistered] = useState(true)
+  const [formValues, setFormValues] = useState<FormValues>({email: '', password: '', confirmPassword: ''});
+  
+  const handleChangeIsRegistered = (): void => {
+    setIsRegistered(!isRegistered)
+  }
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const {name, value} = event.target;
+    setFormValues((prev) => ({...prev, [name]: value}));
+  }
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    console.log('email: ', formValues.email);
+    console.log('password: ', formValues.password);
+  }
+
+  return (
+    <div>
+      {isRegistered === true ? 
```

Comment:
you can do `!!isRegistered` to cast it directly to a boolean value

Reply:
like that one!